### ISSUE TITLE
implementação de Demurage

### ISSUE THREAD

**Issue by ypassy** [evertonfraga/prospera#8]
Acredito que uma taxa aplicada no Prosper parado é mais eficiente para promover a circulação mdo que o mecanismo de inflação.

**Comment by evertonfraga** [comment]
Olá Yoav,

realmente, Prosper parado não é interessante pro sistema mesmo. No entanto tenho uma consideração sobre o demurage: se eu sou uma pessoa que quer apenas acumular Prosper e não ser influenciado pela taxa, eu poderia ter duas contas e ficar transferindo Prosper periodicamente entre elas. 

Um ponto positivo da inflação é que ela recai sobre todos.

**Comment by evertonfraga** [comment]
Como é inviável que a PCD distribua ether em escala (digamos, 10.000 pessoas), no futuro as pessoas terão que buscar seus ethers na PCD com um prazo de validade. Expirado o prazo, esse ether fica disponível pra distribuição seguinte, com regra proporcional pra todos os outros possuidores de PRSP. 

Isso faz com que:
1) O prosper da pessoa inativa não tenha muita utilidade.
2) caso ela não queira mais interagir, é melhor que o venda. 

Nesse jeito não tem taxa, mas tem incentivo a ser ativo na comunidade.

**Comment by ypassy** [comment]
Everton,
pela lição que o Dão nos deu a maioria das pessoas não vão retirar seus Ethers e consequentemente vai acumular um saldo grande atraindo os ladrões. Penso que o PCD seria para iniciar o projeto e ganhar confiança um tipo de "onbording". Uma vez que o projeto existe e está ganhando musculatura a função crowdfunding se torna secundária e a função meio de troca se torna mais  o principal.

**Comment by evertonfraga** [comment]
Concordo plenamente. Vou explicar uma frase do comentário anterior e vai ficar mais claro:

> "Expirado o prazo, esse ether fica disponível pra distribuição seguinte, com regra proporcional pra todos os outros possuidores de PRSP."

Dinheiro parado é muito ruim mesmo. Podemos implementar regras que caso uma pessoa não retire seu ETH, ele será redividido na partilha do período seguinte. 

Deste modo, a PCD acumularia apenas a % dos PRSP de pessoas inativas em ETH, e sempre redistribuindo o restante.
